libs/providers/go-feature-flag/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@openfeature/go-feature-flag-provider",
-  "version": "0.5.0",
+  "version": "0.4.0",
```

Comment:
I am not sure to understand why we are downgrading the version 🤔

Reply:
Just a merge issue. The old PR was VERY out of date, so I had to resolve a lot of conflicts, I knew I missed a few, which is why this is still a draft. Thanks for finding this, I will take a close look tomorrow.

Reply:
@thomaspoignant I think I've corrected all the confict-related issues.

I still have some questions about the deps used here, I will follow up with @james-milligan about that.